README.md

```diff
@@ -21,11 +21,12 @@ Add the module to your Terraform resources:
 
 ```hcl
 module "rds-postgres-password" {
-  source    = "github.com/gearnode/terraform-kubernetes-get-secret?ref=v0.2.0"
+  source    = "github.com/gearnode/terraform-kubernetes-get-secret?ref=v0.3.0"
```

Comment:
We should replace this with the internal link

Reply:
yes I'm gonna do it in another PR just for the doc